index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
please link your CSS file in the correct manner to be applied to your project.
change the name and the directory of the file to be adequate to the current location and name of your CSS sheet